installer/templates/phx_umbrella/apps/app_name_web/README.md

```diff
@@ -3,7 +3,7 @@
 To start your Phoenix app:
 
   * Install dependencies with `mix deps.get`
-  * Create and migrate your database with `mix ecto.create && mix ecto.migrate`
+  * Setup your database with `mix ecto.setup`
```

Comment:
in umbrella, `ecto.setup` alias is defined in the domain logic app and aliases aren't shared between apps, so this does not work out of the box